resources/metavariable-grammars/php-common-metavariable-grammar.js

```diff
@@ -318,7 +323,7 @@ module.exports = function defineGrammar(dialect) {
       enum_case: $ => seq(
         optional(field('attributes', $.attribute_list)),
         keyword('case'),
-        field('name', choice($.name, $.grit_metavariable)),
```

Comment:
All these removals seem wrong - what's the reason?

Reply:
I changed the `$.name` rule to `choice($._name, $.grit_metavariable)` so the choice is no longer necessary everywhere else. 
https://github.com/getgrit/gritql/pull/146/commits/aa4017ca89fe592d9a6c5df19e747bc53fcd7618#diff-d8bdb1fe5a96b8d2e4b8ad3756d3a053d4103b57ae16ad83f8ed2f7a84d33b90L1562-R1566